src/Components/About.css

```diff
@@ -1,21 +1,32 @@
-
 .about-section {
-    width: 99%;
-    box-sizing: border-box;
-    padding: 8px;
-    border: 1px solid #ccc; 
-    margin-top: 16px;
+  width: 99%;
+  box-sizing: border-box;
+  padding: 8px;
+  border: 1px solid #ccc;
+  margin-top: 16px;
 }
 
 .about-section h3 {
-    font-size: 1.5rem;
-    font-weight: bold;
+  font-size: 1.5rem;
+  font-weight: bold;
 }
 
 .about-section p {
-    font-size: 1rem;
-    line-height: 1.6;
-    color: #555;
+  font-size: 1rem;
+  line-height: 1.6;
+  color: #555;
 }
 
-
+.about-section textarea {
+  width: 100%;
+  padding: 12px;
+  margin-top: 10px;
+  font-size: 1rem;
```

Comment:
rem or px. Using different units will violates the consistency. Either go with px or rem. Update it in the whole application

Reply:
fixed